indigo/indigo/src/main/scala/indigo/platform/events/WorldEvents.scala

```diff
@@ -53,7 +55,8 @@ object WorldEvents {
       globalEventStream.pushGlobalEvent(
         MouseEvent.MouseDown(
           absoluteCoordsX(e.clientX) / magnification,
-          absoluteCoordsY(e.clientY) / magnification
+          absoluteCoordsY(e.clientY) / magnification,
+          MouseButton.fromOrdinal(e.button)
```

Comment:
If you had a fancy mouse with more than 5 buttons, this would result in a `NoSuchElementException`. Could we add a similar function to the `MouseButton` companion that checks the range is valid first? Ideally, I feel like hitting a funny button would do nothing (or act like a left click?) rather than blow up. What do you think?